project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -5,15 +5,19 @@
 To establish and administer a fund for members of the OpenJS Foundation to travel
 and spread knowledge about and support the Javascript ecosystem and the Foundation.
 
-### [Current Year Fund](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/2020.md)
+### [Current Year Fund](https://github.com/openjs*foundation/cross*project*council/blob/master/TRAVEL_FUND/2020.md)
```

Comment:
this seems to be a problem throughout the PR. you may want to revert everything and just re-apply the desired changes, without doing a global replace of `-` to `*`?

Reply:
Thank you I have already fixed those in the last commit, i went through the whole file and fixed all the places where there were issues